pkg/capabilities/capabilities.go

```diff
@@ -29,9 +151,97 @@ func Lookup(engine, version string) (*ast.Capabilities, error) {
 	}
 }
 
+func lookupFileURL(parsedURL *url.URL) (*ast.Capabilities, error) {
+	fd, err := os.Open(parsedURL.Path)
+	if err != nil {
+		return nil, err
+	}
+
+	return ast.LoadCapabilitiesJSON(fd)
+}
+
+func lookupWebURL(parsedURL *url.URL) (*ast.Capabilities, error) {
+	req, err := http.NewRequest(http.MethodGet, parsedURL.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := http.DefaultClient.Do(req)
```

Comment:
[nit] Since we do nothing complex here, I think we can use the convenience method: `resp, err := http.Get(parsedURL.String())`

Reply:
I was not aware of that method, but that would be a nice simplification.

Reply:
0527f06